zebra/rt_netlink.c

```diff
@@ -1491,6 +1491,11 @@ static int netlink_route_multipath(int cmd, struct zebra_dplane_ctx *ctx)
 		char buf[NL_PKT_BUF_SIZE];
 	} req;
 
+	if (skip_kernel_install) {
+		zlog_debug("skip installing routes into kernel.");
```

Comment:
Please fix this unguarded debug.  This is going to create an ungodly amount of log file spew